lsp-mode.el

```diff
@@ -7422,7 +7422,7 @@ JavaScript file, tsserver.js (the *.js is required for Windows)."
     (-map (-compose #'symbol-name #'lsp--client-server-id) it)
     (format "%s" it)
     (propertize it 'face 'success)
-    (format "Installing following servers: %s" it)
+    (format " Installing following servers: %s" it)
```

Comment:
Any reason why adding this empty space?

Reply:
This is in mode line, and without the space, it's mangled with previous term.